api/templates/contact_template.txt

```diff
@@ -0,0 +1,3 @@
+Hola, la teva tenda ha rebut la comanda: {{wcfmmp_host}}area-privada/orders-details/{{wc_order_id}}
+
+Adjuntem el PDF per a imprimir i pegar al paquet a enviar.
```

Comment:
we should I18n-ize this content. I guess it's easy with Django.

Reply:
Done in last commits.